blazor/datagrid/foreignkey-column.md

```diff
@@ -0,0 +1,846 @@
+        {
+            EmployeeID = employeeID;
+            FirstName = firstName;
+        }
+
+        public static List<EmployeeData> GetAllRecords()
+        {
+            if (Employees.Count() == 0)
+            {
+                int code = 10;
+                for (int i = 1; i < 2; i++)
+                {
+                    Employees.Add(new EmployeeData( 1, "Nancy"));
+                    Employees.Add(new EmployeeData( 2, "Andrew"));
+                    Employees.Add(new EmployeeData( 3, "Janet"));
+                    Employees.Add(new EmployeeData( 4, "Nancy"));
+                    Employees.Add(new EmployeeData( 5, "Margaret"));
+                    Employees.Add(new EmployeeData( 6, "Steven"));
+                    Employees.Add(new EmployeeData( 7, "Janet"));
+                    Employees.Add(new EmployeeData( 8, "Andrew"));
+                    Employees.Add(new EmployeeData(9, "Nancy"));
+                    code += 5;
+                }
+            }
+            return Employees;
+        }
+        public int? EmployeeID { get; set; }
+        public string FirstName { get; set; }
+    }
+    public class OrderData
+    {
+        public static List<OrderData> Orders = new List<OrderData>();        
+       
+        public OrderData()
+        {
+
+        }
+        public OrderData(int? OrderID, int? EmployeeID, string ShipCity, double? Freight)
+        {
+           this.OrderID = OrderID;
+           this.EmployeeID = EmployeeID;
+           this.ShipCity = ShipCity;
+           this.Freight = Freight;            
+        }
+        public static List<OrderData> GetAllRecords()
+        {
+            if (Orders.Count() == 0)
+            {
+                int code = 10;
+                for (int i = 1; i < 2; i++)
+                {
+                    Orders.Add(new OrderData(10248,1, "Reims", 32.18));
+                    Orders.Add(new OrderData(10249,2, "Münster",33.33));
+                    Orders.Add(new OrderData(10250,3, "Rio de Janeiro",12.35));
+                    Orders.Add(new OrderData(10251,4, "Reims", 22.65));
+                    Orders.Add(new OrderData(10252,5, "Lyon", 63.43));
+                    Orders.Add(new OrderData(10253,6, "Charleroi",56.98));
+                    Orders.Add(new OrderData(10254,7, "Rio de Janeiro", 45.65));
+                    Orders.Add(new OrderData(10255,8, "Münster", 11.13));
+                    Orders.Add(new OrderData(10256,9, "Reims", 87.59));
+                    code += 5;
+                }
+            }
+            return Orders;
+        }
+        public int? OrderID { get; set; }
+        public int? EmployeeID { get; set; }
+        public string ShipCity { get; set; }
+        public double? Freight { get; set; }
+    }
+{% endhighlight %}
+{% endtabs %}
+
+{% previewsample "https://blazorplayground.syncfusion.com/embed/VXBAiiZngvqDDFvZ?appbar=false&editor=false&result=true&errorlist=false&theme=bootstrap5" %}
```

Comment:
sample is not working. once value is filtered. choosing the All option will clear the Grid